frontend/src/lib/components/ServiceRequestPreview.svelte

```diff
@@ -0,0 +1,107 @@
+<script lang='ts'>
+	import messages from '$media/messages.json'
+	import { goto } from '$app/navigation';
+	import { Badge, Card } from 'stwui';
+	import { Dropdown } from 'stwui';
+	import type { GetServiceRequestsResponse, ServiceRequest } from '$lib/services/Libre311/Libre311';
+	import dropDownIcon from '$lib/assets/ellipsis-vertical.svg';
+
+	export let serviceRequest: GetServiceRequestsResponse;
+
+	let visible: Boolean = false;
```

Comment:
This is super minor but 99% of the time you will want to use the `boolean` type.  So it should be:
`let visible: boolean = false;`

The uppercase variant is an Object while the lower case is a primitive.  I'm not sure why it exists in JS.  In Java you have to use the two at different times but in JS you can almost always use the primitive one.